src/test/resources/com/puppycrawl/tools/checkstyle/checks/hexliteralcase/InputHexLiteralCaseCheck.java

```diff
@@ -0,0 +1,18 @@
+/*
+HexLiteralCase
+
+
+*/
+
+package resources.com.puppycrawl.tools.checkstyle.checks.hexliteralcase;
+
+public class InputHexLiteralCaseCheck {
+    int i =  0xabc;  // violation  'Should use uppercase hexadecimal letters.'
+    int a = 0xABC;
+    int k = 0XCAFEBAB1;
+    long d = 0Xf123L; // violation  'Should use uppercase hexadecimal letters.'
+    int binaryValue = 0b1010;
```

Comment:
For each line with violation please add kind of same line but without violation, that would be result of what vis expected refactoring from violated